scripts/release.kts

```diff
@@ -0,0 +1,82 @@
+/**
+ * Outputs the bash script that uploads packages to Bintray.
+ *
+ * This script assumes that all distrbution packages have been downloaded and unzipped in one directory.
+ * For example, all packages are in the "dist" directory:
+ *
+ * dist
+ *  ├── sentry-3.1.2-SNAPSHOT
+ *  ├── sentry-android-3.1.2-SNAPSHOT
+ *  ├── sentry-android-core-3.1.2-SNAPSHOT
+ *  ├── sentry-android-ndk-3.1.2-SNAPSHOT
+ *  ├── sentry-android-timber-3.1.2-SNAPSHOT
+ *  ├── sentry-log4j2-3.1.2-SNAPSHOT
+ *  ├── sentry-logback-3.1.2-SNAPSHOT
+ *  ├── sentry-servlet-3.1.2-SNAPSHOT
+ *  ├── sentry-spring-3.1.2-SNAPSHOT
+ *  └── sentry-spring-boot-starter-3.1.2-SNAPSHOT
+ *
+ * To execute the script two environment variables that are used by Maven have to be present: BINTRAY_USERNAME, BINTRAY_API_KEY
+ *
+ * Example usage (assuming that the script is executed from the `<project-root>/scripts` directory and the distribution files are in `<project-root>/dist`):
+ * $ kotlinc -script release.kts -- -d ../dist -s  -repositoryUrl https://api.bintray.com/maven/sentry/sentry-java/sentry-java/ | sh
```

Comment:
can we have a compiled CLI of it? otherwise, it'll require `kotlinc` in the path, but fine with it for now.

also, important to note that `repositoryUrl` changes per project type (Android or Java)

Reply:
Added two properties for both repositories. We could have compiled but I thought it's just more convenient to have a script in sources..?

Reply:
This will ultimately run on a service so we can make kotlinc be a install requirement, prob best to be editable/script at this point

Reply:
yeah I just wanted to make it easier, even myself didn't have kotlinc on the system env path, I've always used Gradle so I needed to https://kotlinlang.org/docs/tutorials/command-line.html
but anyway, via `brew` is 2s so I'm in for it